orga/app/routes/join/signup.js

```diff
@@ -0,0 +1,45 @@
+import Route from '@ember/routing/route';
+import { service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+export default class SignupRoute extends Route {
+  @service router;
+  @service session;
+  @service store;
+
+  routeIfAlreadyAuthenticated = 'join-when-authenticated';
+  @tracked isInvitationCancelled = false;
+  @tracked hasInvitationAlreadyBeenAccepted = false;
```

Comment:
Petite suggestion de style pour organiser les différents attributs de la classe : 
```suggestion
  @service router;
  @service session;
  @service store;

  @tracked isInvitationCancelled = false;
  @tracked hasInvitationAlreadyBeenAccepted = false;
  
    routeIfAlreadyAuthenticated = 'join-when-authenticated';
```